db/migrate/20210331165936_move_auto_silence_first_post_regex_to_watched_words.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class MoveAutoSilenceFirstPostRegexToWatchedWords < ActiveRecord::Migration[6.0]
+  def up
+    execute <<~SQL
```

Comment:
Since this setting is a regexp, shouldn't we also enable the `watched words regular expressions` as well?

We might want to do it only if there was a value in the first place though.